ocsmesh/cli/remesh_by_shape_factor.py

```diff
@@ -1,82 +1,102 @@
 #!/usr/bin/env python
-from pathlib import Path
-from copy import deepcopy
 import logging
 import sys
+from copy import deepcopy
+from pathlib import Path
 
 import geopandas as gpd
+import jigsawpy
 import numpy as np
+from pyproj import Transformer
 from shapely.geometry import MultiPolygon, Polygon
 from shapely.ops import transform
-import jigsawpy
-from pyproj import Transformer
 
-from ocsmesh import Raster, Geom, Hfun, Mesh
-from ocsmesh import utils
+from ocsmesh import Geom, Hfun, Mesh, Raster, utils
 
 logging.basicConfig(
     stream=sys.stdout,
-    format='%(asctime)s,%(msecs)d %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s',
-    datefmt='%Y-%m-%d:%H:%M:%S'
-    )
-#logging.getLogger().setLevel(logging.DEBUG)
-#logging.getLogger().setLevel(logging.INFO)
+    format="%(asctime)s,%(msecs)d %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s",
+    datefmt="%Y-%m-%d:%H:%M:%S",
+)
+# logging.getLogger().setLevel(logging.DEBUG)
+# logging.getLogger().setLevel(logging.INFO)
 
 _logger = logging.getLogger(__name__)
 
 # Enable KML driver
-gpd.io.file.fiona.drvsupport.supported_drivers['KML'] = 'rw'
+gpd.io.file.fiona.drvsupport.supported_drivers["KML"] = "rw"
 
-class RemeshByShape:
 
+class RemeshByShape:
     @property
     def script_name(self):
-        return 'remesh_by_shape'
+        return "remesh_by_shape"
 
     def __init__(self, sub_parser):
 
         this_parser = sub_parser.add_parser(self.script_name)
 
-        this_parser.add_argument('--in-crs', default='EPSG:4326')
-        this_parser.add_argument('--shape', type=str)
+        this_parser.add_argument("--in-crs", default="EPSG:4326")
+        this_parser.add_argument("--shape", type=str)
 
-        this_parser.add_argument('-u', '--upstream', action='store_true')
+        this_parser.add_argument("-u", "--upstream", action="store_true")
         this_parser.add_argument(
-            '--cutoff', default=-250, type=float,
-            help="Refinement cutoff depth in meters (positive up)")
-        this_parser.add_argument('--factor', default=2, type=float)
+            "--cutoff",
+            default=-250,
+            type=float,
+            help="Refinement cutoff depth in meters (positive up)",
+        )
+        this_parser.add_argument("--factor", default=2, type=float)
         this_parser.add_argument(
-            '--contour',
-            action='append', nargs='+', type=float, dest='contours',
-            metavar='CONTOUR_DEFN', default=[],
+            "--contour",
+            action="append",
+            nargs="+",
+            type=float,
+            dest="contours",
+            metavar="CONTOUR_DEFN",
+            default=[],
```

Comment:
I think it's too much to spread every argument to a newline. Is there any way to control linter not to do this?